__tests__/news/news.mutation.test.js

```diff
@@ -0,0 +1,260 @@
+      __typename: 'Author',
+      name: [
+        { __typename: 'Language', lang: 'uk', value: 'a sd' },
+        { __typename: 'Language', lang: 'eng', value: 'a sd' },
+      ],
+    });
+  });
+  test('#2 update news', async () => {
+    const res = await client.mutate({
+      mutation: gql`
+        mutation($id: ID!) {
+          updateNews(
+            id: $id
+            news: {
+              text: [
+                { lang: "uk", value: "u p d a t e N1 d a s d" }
+                { lang: "eng", value: "update dN1 a s d" }
+              ]
+              author: {
+                name: [
+                  { lang: "uk", value: "updated sd" }
+                  { lang: "eng", value: "updated sd" }
+                ]
+              }
+            }
+          ) {
+            ... on News {
+              _id
+              title {
+                lang
+                value
+              }
+              text {
+                lang
+                value
+              }
+              author {
+                name {
+                  lang
+                  value
+                }
+              }
+            }
+            ... on Error {
+              message
+            }
+          }
+        }
+      `,
+      variables: { id: NEWS_ID },
+    });
+
+    expect(res.data.updateNews.text).toBeInstanceOf(Array);
+    expect(res.data.updateNews).toHaveProperty('text', [
+      { __typename: 'Language', lang: 'uk', value: 'u p d a t e N1 d a s d' },
+      { __typename: 'Language', lang: 'eng', value: 'update dN1 a s d' },
+    ]);
+    expect(res.data.updateNews.author).toBeInstanceOf(Object);
+    expect(res.data.updateNews).toHaveProperty('author', {
+      __typename: 'Author',
+      name: [
+        { __typename: 'Language', lang: 'uk', value: 'updated sd' },
+        { __typename: 'Language', lang: 'eng', value: 'updated sd' },
+      ],
+    });
+  });
+
+  test('#3 update not existing news should return error', async () => {
+    const res = await client.mutate({
+      mutation: gql`
+        mutation($id: ID!) {
+          updateNews(
+            id: $id
+            news: {
+              text: [
+                { lang: "uk", value: "u p d a t e N1 d a s d" }
+                { lang: "eng", value: "update dN1 a s d" }
+              ]
+            }
+          ) {
+            ... on News {
+              _id
+              text {
+                lang
+                value
+              }
+              title {
+                lang
+                value
+              }
+              author {
+                name {
+                  lang
+                  value
+                }
+              }
+            }
+            ... on Error {
+              message
+              statusCode
+            }
+          }
+        }
+      `,
+      variables: { id: NEWS_DOES_NOT_EXIST_ID },
+    });
+    expect(res.data.updateNews).toHaveProperty('message');
```

Comment:
Please, check the actual value of the `message` and `statusCode`, I mean numeric value and the error message text

Reply:
fixed